lib/cocoapods-playgrounds/workspace.rb

```diff
@@ -152,31 +152,23 @@ def generate_project
                                   @platform,
                                   @deployment_target)
       target.build_configurations.each do |config|
-        config.build_settings['ASSETCATALOG_COMPILER_LAUNCHIMAGE_NAME'] = 'LaunchImage'
-        config.build_settings['CODE_SIGN_IDENTITY[sdk=iphoneos*]'] = ''
-        config.build_settings['CODE_SIGNING_ALLOWED'] = 'NO'
-        config.build_settings['CODE_SIGNING_REQUIRED'] = 'NO'
-        config.build_settings['DEFINES_MODULE'] = 'NO'
-        config.build_settings['EMBEDDED_CONTENT_CONTAINS_SWIFT'] = 'NO'
+        update_build_settings(config)
       end
 
       # TODO: Should be at the root of the project
       project.new_file("#{names.first}.playground")
       project.save
     end
 
-    def generate_swift_code(path)
-      File.open(path + 'Contents.swift', 'w') do |f|
-        f.write("//: Please build the scheme '#{target_name}' first\n")
-        f.write("import PlaygroundSupport\n")
-        f.write("PlaygroundPage.current.needsIndefiniteExecution = true\n\n")
-        unless potential_cartfile
-          names.each do |name|
-            f.write("import #{name}\n")
-          end
-          f.write("\n")
-        end
-      end
+    def update_build_settings(config)
+      config.build_settings['ASSETCATALOG_COMPILER_LAUNCHIMAGE_NAME'] = 'LaunchImage'
+      config.build_settings['CODE_SIGN_IDENTITY[sdk=iphoneos*]'] = ''
+      config.build_settings['CODE_SIGNING_ALLOWED'] = 'NO'
+      config.build_settings['CODE_SIGNING_REQUIRED'] = 'NO'
+      config.build_settings['DEFINES_MODULE'] = 'NO'
+      config.build_settings['EMBEDDED_CONTENT_CONTAINS_SWIFT'] = 'NO'
+      # TODO: define swift version correctly
```

Comment:
TODO: fix in this PR

Reply:
fixes #48

Reply:
this works as a stopgap for common cases. filed #59 to adaptively pick the right version, allow a command line flag, and/or other appropriate fixes.